data/sample_panel.csv

```diff
@@ -1,40 +1,40 @@
-32,0.41022099812651525,Cytokeratin17,0.11022099812651526,0.41022099812651525
-33,0.5538475546270879,COL1A1,0.2538475546270879,0.5538475546270879
-34,0.34100502081924133,H3K27me3,0.041005020819241345,0.34100502081924133
-35,0.957781589809816,CD57,0.657781589809816,0.957781589809816
-36,0.7235428820159893,H3K9ac,0.4235428820159893,0.7235428820159893
-37,0.01630130408749564,Ki67,-0.28369869591250435,0.01630130408749564
-38,0.37589300123288616,HLA1 class ABC,0.07589300123288617,0.37589300123288616
+Mass,Target,Start,Stop
+0.6166032817280968,Au,0.31660328172809676,0.6166032817280968
+0.7361271393405203,Chymase,0.43612713934052033,0.7361271393405203
+0.3882306801433095,SMA,0.08823068014330954,0.3882306801433095
+0.21260791446689753,Vimentin,-0.08739208553310246,0.21260791446689753
+0.008343697274465045,LAG3,-0.29165630272553494,0.008343697274465045
+0.1024505300403129,CD4,-0.19754946995968709,0.1024505300403129
+0.09538553523509907,CD69,-0.20461446476490092,0.09538553523509907
+0.4677381510905112,FAP,0.1677381510905112,0.4677381510905112
+0.02203986896035326,FOXP3,-0.27796013103964673,0.02203986896035326
+0.19731202380086854,PD1,-0.10268797619913145,0.19731202380086854
+0.04031172518395432,CD31,-0.25968827481604567,0.04031172518395432
+0.6293475980907576,Biotin,0.3293475980907576,0.6293475980907576
+0.9216318061684895,Ecadherin,0.6216318061684896,0.9216318061684895
+0.4320585151087698,CD56,0.13205851510876981,0.4320585151087698
+0.8549039323425051,CD38,0.5549039323425051,0.8549039323425051
+0.4320891324728383,TCF1 TCF7,0.13208913247283832,0.4320891324728383
+0.10966691868933631,TBET,-0.19033308131066368,0.10966691868933631
+0.824365503762223,CD45RB,0.524365503762223,0.824365503762223
+0.46899467150493834,CD68,0.16899467150493835,0.46899467150493834
+0.8960437346678288,CD11c,0.5960437346678289,0.8960437346678288
+0.2069459497701247,CD8,-0.09305405022987528,0.2069459497701247
+0.8170810324955535,CD3e,0.5170810324955535,0.8170810324955535
+0.5822900443249691,IDO1,0.2822900443249691,0.5822900443249691
+0.08433591073041669,CD45RO,-0.2156640892695833,0.08433591073041669
+0.004651565844378536,TIM-3,-0.29534843415562145,0.004651565844378536
+0.7833558782673933,CD163,0.4833558782673933,0.7833558782673933
+0.5774063357331318,CD20,0.27740633573313184,0.5774063357331318
+0.4774159051577329,FN1,0.17741590515773292,0.4774159051577329
+0.3092266809280192,Glut1,0.009226680928019204,0.3092266809280192
+0.08037466083170608,HLADR,-0.2196253391682939,0.08037466083170608
+0.026329017704782465,CD14,-0.2736709822952175,0.026329017704782465
+0.43767397644186756,CD45,0.13767397644186757,0.43767397644186756
+0.41022099812651525,Cytokeratin17,0.11022099812651526,0.41022099812651525
+0.5538475546270879,COL1A1,0.2538475546270879,0.5538475546270879
+0.34100502081924133,H3K27me3,0.041005020819241345,0.34100502081924133
+0.957781589809816,CD57,0.657781589809816,0.957781589809816
+0.7235428820159893,H3K9ac,0.4235428820159893,0.7235428820159893
```

Comment:
Are all these difs okay? Looks like rownames getting removed?

Reply:
Yeah it's the indices. The output from panel_utils.convert_panel() doesn't have an index column so technically if someone tried to used the current panel in a notebook as a test it would error out.